src/main/java/com/google/cloud/pontem/CloudSpannerDatabaseBackup.java

```diff
@@ -153,6 +153,13 @@
 
     void setOutputFolder(String value);
 
+    /** Whether to overwrite existing GCS file contents (if any contents exist). */
+    @Description("Whether to overwrite GCS file contents.")
+    @Default.Boolean(false)
+    Boolean getWhetherToOverwriteGcsFileBackup();
```

Comment:
Up to you but, it seems that shouldOverwrite() is clearer than whetherToOverwrite() to denote that this is a boolean and how it's expected to work.